components/dashboards-web-component/src/viewer/components/DashboardReportGenerationCard.jsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Component } from 'react';
+import { Card, CardHeader, CardActions, RaisedButton, List, ListItem, SelectField, MenuItem } from 'material-ui';
+import DeleteIcon from 'material-ui/svg-icons/action/delete';
+import ReportGenerationButton from './ReportGenerationButton';
+import html2canvas from 'html2canvas';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+const pageSizes = ['A4', 'Letter', 'A3'];
+const orientations = ['Landscape','Portrait'];
+
+export default class DashboardReportGenerationCard extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: false,
+            pageList: [],
+            pageSize: 'A4',
+            orientation:'Landscape'
+        };
+
+        this.handleCardClick = this.handleCardClick.bind(this);
+        this.capturePage = this.capturePage.bind(this);
+        this.removePage = this.removePage.bind(this);
+    }
+
+    render() {
+        return (
+            <Card
+                style={{ margin: 10 }}
+                expanded={this.state.expanded}
+                onExpandChange={this.handleCardClick}
+            >
+                <CardHeader title={<FormattedMessage id='dashboardReport.title' defaultMessage='Export Dashboard As Pdf'/>} actAsExpander textStyle={{ paddingRight: '0px' }} />
```

Comment:
use a specific prefix for all the localized strings of your feature. 
Ex : report.generation.export.button
       report.generation.page.capture.button